keps/sig-instrumentation/2831-kubelet-tracing/README.md

```diff
@@ -319,20 +340,21 @@ _This section must be completed when targeting beta graduation to a release._
 
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
-  Upgrades and rollbacks will be tested while feature-gate is experimental
+
+  Yes. These were tested on a 1.27 kind cluster by enabling, disabling, and re-enabling the feature-gate on the kubelet.
```

Comment:
Can you please be a bit more specific on how exactly you did that?

It doesn't have to be as detailed as https://github.com/mimowo/enhancements/blob/9303678d20aef13af6fdb85a98128deb7603cd94/keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs/README.md#were-upgrade-and-rollback-tested-was-the-upgrade-downgrade-upgrade-path-tested but a bit more detail would be useful here.

Reply:
updated to include detailed steps I took